build.gradle

```diff
@@ -30,7 +30,7 @@ ext {
 
     supportLibVersion = '28.0.0'
     playServicesVersion = '15.0.1'
-    workVersion = '1.0.0-alpha11'
+    workVersion = '1.0.0-beta04'
```

Comment:
Beta05 is now also available: https://developer.android.com/jetpack/androidx/releases/work#1.0.0-beta05

Reply:
Release candidate is now available! :) https://developer.android.com/jetpack/androidx/releases/work#1.0.0-rc02

Reply:
We finally have a stable release of work manager 1.0.0

https://developer.android.com/jetpack/androidx/releases/work#1.0.0

Reply:
Amazing, I'll update the PR tomorrow, I'm sorry for the delay!